zerver/lib/actions.py

```diff
@@ -3257,6 +3267,23 @@ def name(sub):
                 state['twenty_four_hour_time'] = event["setting"]
             if event['setting_name'] == 'left_side_userlist':
                 state['left_side_userlist'] = event["setting"]
+        elif event['type'] == "update_global_notifications":
+            if event['notification_name'] == "enable_stream_desktop_notifications":
```

Comment:
Well, I'm not sure -- it isn't done that way in other places in the code... (as far as I could see)